### PR TITLE
feat: animations

### PR DESCRIPTION
**Pull request by Tulun**
1. Hero animations
2. Button / hover effects / navbar (etc) transitions.
3. Project button transitions.
4. Tab content

Screen Shot 2022-11-15 at 5 52 52 PM
Screen Shot 2022-11-15 at 5 52 41 PM
Screen Shot 2022-11-15 at 5 52 33 PM
Screen Shot 2022-11-15 at 5 52 29 PM
Screen Shot 2022-11-15 at 5 52 22 PM
Screen Shot 2022-11-15 at 5 52 16 PM

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -313,6 +313,7 @@ const Input = styled.input`
   font: var(--body-md);
   caret-color: var(--grey-100);
   border: 2px solid transparent;
+  transition: all 0.3s ease-in-out;
```
[the remark this answers is not in the document]
Comment (ryanwolhuter) [reply components/Footer/Footer.tsx:316]: yeah I agree and also I don't recommend using transition all because it can have unintended effects

**Review by ryanwolhuter — APPROVED** [review]
Transitions are all looking sick dude well done!

File: components/Widgets/Widgets.tsx [hunk 2]
```diff
@@ -28,8 +28,8 @@ export const Header = styled.div`
   font: var(--header-lg);
   letter-spacing: -0.01em;
   position: relative;
-  background: -webkit-radial-gradient(center, 50% 50%, red, black);
+  /* background: -webkit-radial-gradient(center, 50% 50%, red, black);
   -webkit-background-clip: text;
-  -webkit-text-fill-color: transparent;
+  -webkit-text-fill-color: transparent; */
```
[the remark this answers is not in the document]
Comment (Tulun) [reply components/Widgets/Widgets.tsx:33]: WIP on a difficult animation.

File: components/Footer/Footer.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tulun) [reply components/Footer/Footer.tsx:316]: moved to a variable.

File: components/Hero/Hero.tsx [hunk 3]
```diff
@@ -47,15 +65,34 @@ function useHero() {
...
-  background-repeat: no-repeat;
-  background-position: center bottom;
+  position: relative;
+  overflow: hidden;
+  &::after {
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-image: url("/assets/hero-bg-img.svg");
```
[the remark this answers is not in the document]
Comment (Tulun) [reply components/Hero/Hero.tsx:88]: Hmm.

So the issue is when I import SVG it considers a component, where this is looking for a string. if you're asking me to do the following: import HeroBG from "public/assets/hero-bg-img.svg";

File: components/Projects/Projects.tsx [hunk 4]
```diff
@@ -231,6 +238,7 @@ const BigProject = styled.div`
   width: 280px;
   height: 280px;
   justify-content: center;
+  transition: all 0.2s ease-in-out;
```
[the remark this answers is not in the document]
Comment (Tulun) [reply components/Projects/Projects.tsx:241]: This animation may be a bit faster yes, but this is an approximation based on the Figma.

**Pull request merged**